includes/_header.htm

```diff
@@ -17,7 +17,7 @@
 		<meta property="og:type" content="website" />
 		<link href="/css/fonts.css" rel="stylesheet">
 		<script src="/js/jquery.min.js"></script>
-		<script defer src="https://use.fontawesome.com/releases/v5.0.8/js/all.js"></script>
+		<script defer src="/js/all.js"></script>
```

Comment:
let's give this file a descriptive name e.g `fontawesome.js`

Reply:
done